docs/modules/traverse.md

```diff
@@ -0,0 +1,47 @@
+# traverse
+
+Traverse and transform objects by visiting every node on a recursive walk.
+
+```js
+var traverse = require('traverse');
```

Comment:
We don't need an example here, only the individual replacements should really have one